### ISSUE TITLE
[BITV]: Button "back" is an icon-button and have no alternative text

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/photos#2239]
![Screenshot from 2024-01-08 18-33-09](https://github.com/nextcloud/photos/assets/6078378/de156f21-e34c-4892-b7a7-7f5c1c0fb205)

`aria-label` have to be there (NcVue only icon button)

**Comment by szaimen** [comment]
The back button seems to be missing now and I cannot open any picture here anymore...
![image](https://github.com/nextcloud/photos/assets/42591237/8d923fe3-cbed-44d8-9fea-254c446d58de)

**Comment by susnux** [comment]
Its position needs to be absolute instead of fixes I guess

**Comment by JuliaKirschenheuter** [comment]
some other regression? I haven't changed position of the button

**Comment by szaimen** [comment]
This is still broken:

https://github.com/nextcloud/photos/assets/42591237/cf993492-0154-4084-bf78-d08deecb365a

**Comment by szaimen** [comment]
Opened https://github.com/nextcloud/photos/issues/2321 for that. This issue here issue seems to be fixed :)
